.github/workflows/countline.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+
+import os
+import glob
+
+#get current working dir, set count, and select file delimiter
+path = os.getcwd()
+#os.chdir(path)
+
+
+#parses through files and saves to a dict
+names={}
```

Comment:
Create a main function that runs

Reply:
Instead of creating class and function, I have created a simple script that ends with 1 if error occurred and 0 for successful execution.